# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/classicube/libdir.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- a/Makefile.2
+++ b/Makefile
@@ -38,7 +38,7 @@ LIBS=-mwindows -lwinmm -limagehlp
 endif

 ifeq ($(PLAT),linux)
-LIBS=-lX11 -lXi -lpthread -lGL -lm -ldl
+LIBS=$(shell pkg-config --libs xi x11) -lpthread -lGL -lm -ldl
 endif

 ifeq ($(PLAT),sunos)
